src/target.rs

```diff
@@ -60,10 +60,19 @@ impl BindTarget for Key {
             Self::PageDown => "PGD".into(),
             Self::PlusEquals => "=".into(),
             Self::Minus => "-".into(),
-            _ => match unsafe { transmute::<_, u64>(discriminant(self)) } {
-                i @ 17..=26 => format!("{}", i - 17),
-                _ => format!("{self:?}"),
-            },
+            _ => {
+                #[cfg(target_family = "wasm")]
+                match unsafe { transmute::<_, u32>(discriminant(self)) } {
```

Comment:
I think it would be better to remove transmute all together and match keys from `Num0` to `Num1` so they format as `0`, `1`, etc. And the rest of the keys can use debug formatting.